lib/CMakeLists.txt

```diff
@@ -283,8 +283,18 @@ if (JRUBY_SUPPORT)
 
     # javah does not atomically write the header file, so parallel builds can
     # read it before it finishes writing if not careful.
+    # JDK versions after 9 don't provide javah. Use javac in these cases
+
+    if(Java_VERSION VERSION_LESS "10")
+	    set(this_JAVAH_COMMAND javah)
```

Comment:
I'm not really a fan of the `this_` style here - we don't use it anywhere else in the CMake code.